CONTRIBUTING.md

```diff
@@ -345,6 +345,25 @@ with clear instructions on how to install the required libraries.
 Furthermore, if your package requires CGO, it MUST be able to compile and operate in a no op mode
 or report a warning back to the collector with a clear error saying CGO is required to work.
 
+### Breaking changes
```

Comment:
What about changing an interface. Adding a func? Do we want to go with optional interface? I don't think there is a way if the interface is used in multiple funcs as argument, because you need to deprecate every one.

Reply:
I don't have the perfect answer, but this question is relevant also for post-v1 and the answer should be the same.

Reply:
Post v1 I think the willingness to pay any cost is a "given" :))